contribution/generator/src/TestGenerator.php

```diff
@@ -53,7 +53,11 @@ public function createTestsFor(
             $method = $this->builderFactory->method($methodName)
                 ->makePublic()
                 ->setReturnType('void')
-                ->setDocComment("/**\n * uuid: {$case->uuid}\n * @testdox {$description}\n */")
+                ->setDocComment("/**\n * uuid: {$case->uuid}\n */")
```

Comment:
My preference would be:

```suggestion
                ->setDocComment("/** uuid: {$case->uuid} */")
```

Reply:
Applied. I find this debatable, personally I usually prefer to avoid one-line doc comment for functions and classes.

Note that I didn't update the files to reflect this change in all the generated files, do you think that I should do so?